Client/Frontend/Strings.swift

```diff
@@ -347,6 +347,54 @@ extension String {
                 comment: "Button title which, when tapped, will allow the user to save valid credit card details.")
         }
 
+        // Remember Card
+        public struct RememberCard {
+            public static let MainTitle = MZLocalizedString(
+                key: "RememberCardTitle.Default",
```

Comment:
I would include the prefix `CreditCard.` in each key. Also make sure the string name is not part of the `RememberCard` suffix. Is there any reasons to add `Default` as well? If not, we can just put the version instead. 
```suggestion
                key: "CreditCard.RememberCard.Title.v115",
```

Similar comment will apply for strings under UpdateCard